### ISSUE TITLE
Enhanced documentation to write native drivers in the Ada programming language.

### ISSUE THREAD

**Issue by ohenley** [kendryte/kendryte-doc-ds#14]
> FEATURE REQUEST

1. Enough low level information/documentation to write native drivers in the Ada programming language for the Kendrite K210. (The community will write the drivers, but we lack some low level details)

1. a) SVD support would be great too. 

### Missing feature

1. What is needed to implement drivers would be the information about the Special Function Registers. That are memory locations that are used to configure the peripherals. The location and size of these registers must be known. Also the meaning of all the bits in them, etc.

1. a) SVD: http://www.keil.com/pack/doc/CMSIS/SVD/html/index.html

### Justification

- At the moment Ada does not support the Kendrite K210.
- Ada/SPARK is a language of choice for embedded computing. The addition of the Kendrite would be awesome!

### Workarounds

None

**Comment by Fabien-Chouteau** [comment]
This would actually be useful for any "alternative" programming language.

**Comment by fb39ca4** [comment]
The SVD file exists.
https://github.com/riscv-rust/k210-pac

**Comment by vmedea** [comment]
> The SVD file exists.

yes, though mind it's a community effort reconstructed from the C SDK structures and comments, which is better than nothing, but i still wish there will be official register documentation at some point
